spec/views/groups/new.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "groups/new", type: :view do
+  before do
+    assign(:group, Group.new(
```

Comment:
Same as in spec/views/groups/edit.html.erb_spec.rb

Reply:
should resolve with eb5b2c7